provider/azure/internal/imageutils/images.go

```diff
@@ -107,11 +175,11 @@ func ubuntuSKU(ctx context.ProviderCallContext, series, stream, location string,
 	for _, img := range result.VirtualMachineImageResourceArray {
 		skuName := *img.Name
 		logger.Debugf("Found Azure SKU Name: %v", skuName)
-		if !strings.HasPrefix(skuName, seriesVersion) {
-			logger.Debugf("ignoring SKU %q (does not match series %q with version %q)", skuName, series, seriesVersion)
+		if !strings.HasPrefix(skuName, desiredSKUPrefix) {
+			logger.Debugf("ignoring SKU %q (does not match base %q)", skuName, base)
```

Comment:
since we are looking at skus with series, let's continue to include series in the message. the original message was ok